doc/lms.mysql

```diff
@@ -169,6 +169,7 @@ DROP TABLE IF EXISTS customerbalances;
 DROP TABLE IF EXISTS comments;
 DROP TABLE IF EXISTS cryptokeys;
 DROP TABLE IF EXISTS tsigkeys;
+DROP TABLE IF EXISTS vlans;
 
```

Comment:
Dlaczego tu na końcu, a definicja gdzieś w miejscu, gdzie to mam sens?
Kolejność usuwania tabel jest istotna - gdy usuwasz tabelę od której zależy tabela **vlans** to będzie problem.

Reply:
Nie wziąłem tego pod uwagę gdyż na codzień w takich sytuacjach używam CASCADE.
Poprawione.